blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='filter-control']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-ff-icon='filter-dropdown-icon']"),
+		      filterContent = filterDropdown.closest("[data-ff='filter']").find("[data-ff='filter-content']");
```

Comment:
That's probably fine, usually I'd try to set the handler on the module itself and use delegation (have fewer listeners as things get more complex, which is always good) but in this case this is probably the simplest way.